hummingbot/strategy/pure_market_making/pure_market_making_order_tracker.pxd

```diff
@@ -0,0 +1,10 @@
+# distutils: language=c++
+
+from hummingbot.strategy.order_tracker import OrderTracker
+from hummingbot.strategy.order_tracker cimport OrderTracker
+
+
+cdef class PureMarketMakingOrderTracker(OrderTracker):
+
+    cdef c_stop_tracking_limit_order(self, object market_pair, str order_id)
```

Comment:
Since both of these functions are already defined in the base class, you don't need to declare them here again. You can just write `pass` here.